network-store-server/src/test/java/com/powsybl/network/store/server/NetworkStoreControllerIT.java

```diff
@@ -887,9 +1154,167 @@ void test() throws Exception {
+                .build();
+        createIdentifiable(twoWT2, "2-windings-transformers");
+
+        deleteIdentifiableList(List.of("TwoWT1", "TwoWT2"), "2-windings-transformers");
+
+        // ThreeWindingsTransformer
+        Resource<ThreeWindingsTransformerAttributes> threeWT1 = Resource.threeWindingsTransformerBuilder()
+                .id("ThreeWT1")
+                .attributes(ThreeWindingsTransformerAttributes.builder()
+                        .name("ThreeWT1")
+                        .leg1(LegAttributes.builder().voltageLevelId("baz").build())
+                        .leg2(LegAttributes.builder().voltageLevelId("baz").build())
+                        .leg3(LegAttributes.builder().voltageLevelId("baz").build())
+                        .build())
+                .build();
+        createIdentifiable(threeWT1, "3-windings-transformers");
+
+        Resource<ThreeWindingsTransformerAttributes> threeWT2 = Resource.threeWindingsTransformerBuilder()
+                .id("ThreeWT2")
+                .attributes(ThreeWindingsTransformerAttributes.builder()
+                        .name("ThreeWT2")
+                        .leg1(LegAttributes.builder().voltageLevelId("baz").build())
+                        .leg2(LegAttributes.builder().voltageLevelId("baz").build())
+                        .leg3(LegAttributes.builder().voltageLevelId("baz").build())
+                        .build())
+                .build();
+        createIdentifiable(threeWT2, "3-windings-transformers");
+
+        deleteIdentifiableList(List.of("ThreeWT2", "ThreeWT1"), "3-windings-transformers");
+
+        // BusbarSection
+        Resource<BusbarSectionAttributes> busBar1 = Resource.busbarSectionBuilder()
+                .id("busBar1")
+                .attributes(BusbarSectionAttributes.builder().name("busBar1").build())
+                .build();
+        createIdentifiable(busBar1, "busbar-sections");
+
+        Resource<BusbarSectionAttributes> busBar2 = Resource.busbarSectionBuilder()
+                .id("busBar2")
+                .attributes(BusbarSectionAttributes.builder().name("busBar2").build())
+                .build();
+        createIdentifiable(busBar2, "busbar-sections");
+
+        deleteIdentifiableList(List.of("busBar2", "busBar1"), "busbar-sections");
+
+        // ConfiguredBus
+        Resource<ConfiguredBusAttributes> bus1 = Resource.configuredBusBuilder()
+                .id("bus1")
+                .attributes(ConfiguredBusAttributes.builder().name("bus1").build())
+                .build();
+        createIdentifiable(bus1, "configured-buses");
+
+        Resource<ConfiguredBusAttributes> bus2 = Resource.configuredBusBuilder()
+                .id("bus2")
+                .attributes(ConfiguredBusAttributes.builder().name("bus2").build())
+                .build();
+        createIdentifiable(bus2, "configured-buses");
+
+        deleteIdentifiableList(List.of("bus1", "bus2"), "configured-buses");
```

Comment:
These tests are already very big/complex, can we add the deleteIdentifiables tests in a separate tests instead of adding to exisiting one?
Leave the existing one as is juste replacing with list where necessary and add a separate to test list delete.

Reply:
done